sdk/keyvault/azkeys/client.go

```diff
@@ -90,7 +90,7 @@ type CreateKeyOptions struct {
 	PublicExponent *int32 `json:"public_exponent,omitempty"`
 
 	// Tags is application specific metadata in the form of key-value pairs.
-	Tags map[string]string `json:"tags,omitempty"`
+	Tags map[string]*string `json:"tags,omitempty"`
```

Comment:
Is this consistent with other SDKs? Tag dictionaries on Azure can't even have an empty/null value so this should never be nil nor could it ever be empty which I know is the zero value if using only a single return from `map[]` instead of the 2 value return. Just seems the indirection here is unnecessary.

Reply:
Yes. This is how autorest.go models it, so all our management plane libraries use this type for tags. I agree the pattern is awkward, but it enables us to accommodate future null elements, however unlikely they may be, without a breaking change.

Reply:
Okay. I wouldn't expect it in this PR, but just wanted to make sure you were aware. Sorry about that. ;)